package.json

```
@@ -15,15 +15,15 @@
     "lint": "eslint .",
     "prettier:check": "prettier --check '{src,pages,cypress}/**/*.{js,ts,tsx}'",
     "prettier:write": "prettier --write '{src,pages,cypress}/**/*.{js,ts,tsx}'",
-    "check": "yarn lint && yarn prettier:check && yarn build",
+    "check": "yarn lint && yarn prettier:check && yarn tsc --noEmit",
     "release": "standard-version -a",
     "release:first": "standard-version -a --first-release",
     "release:minor": "standard-version -a --release-as minor",
     "release:major": "standard-version -a --release-as major",
     "release:patch": "standard-version -a --release-as patch",
     "postinstall": "husky install",
     "post-commit": "git status",
-    "pre-commit": "yarn prettier:check && yarn lint",
+    "pre-commit": "yarn prettier:check && yarn lint && yarn tsc --noEmit",
```

Since they are the same
```suggestion
    "pre-commit": "yarn check",
```